content/en/docs/Installation/installation-guide/install-with-helm.md

```diff
@@ -69,6 +69,7 @@ command:
 $ helm install \
     --set cr.create=true \
     --set cr.namespace=istio-system \
+    --set cr.auth.strategy="anonymous" \
```

Comment:
This is my fault. I did not properly review this PR.

This is wrong and does nothing to affect the strategy. It should have been:

```
    --set cr.spec.auth.strategy="anonymous" \
```

(note the missing `.spec`)